website/snippets/_mesh-cycle-detection.md

```diff
@@ -1,14 +1,9 @@
-Currently, the default behavior for "project" dependencies enforces that these relationships only go in one direction, meaning that the `jaffle_finance` project could not add a new model that depends, on any public models produced by the `jaffle_marketing` project. dbt will check for cycles across projects and raise errors if any are detected.
+You can enable bidirectional dependencies across projects so these relationships can go in either direction, meaning that the `jaffle_finance` project can add a new model that depends on any public models produced by the `jaffle_marketing` project, so long as the new dependency doesn't introduce any node-level cycles. dbt checks for cycles across projects and raises errors if any are detected.
```

Comment:
Based on the placement of this snippet, that first sentence may not be descriptive enough.  What relationships are we alluding to?

We're also using specific examples that aren't really anywhere else on the Mesh-5-faqs and Mesh-3-Structures pages that use the snippet. It might be better to use the same generic examples we use down on lines 9-12

Reply:
that's fair! open to any suggs you may have? 

also, is this a blocker?